benchmarking/README.md

```diff
@@ -101,15 +101,21 @@ into a CSV file for analysis.
 For example (from the `FirstOrderLp.jl` root directory),
 
 ```sh
-$ julia --project=scripts scripts/solve_qp.jl \
---instance_path test/trivial_lp_model.mps --method pdhg \
---output_dir /tmp/first_order_lp_solve
-$ julia --project=scripts scripts/solve_lp_external.jl \
---instance_path test/trivial_lp_model.mps --solver scs-indirect \
---tolerance 1e-7 --output_dir /tmp/scs_solve
+$ benchmarking/collect_lp_benchmark.sh /tmp ~/lp_benchmark
+$ for INSTANCE in nug08-3rd qap15
+do
+  julia --project=scripts scripts/solve_qp.jl \
+    --instance_path ~/lp_benchmark/${INSTANCE}.mps.gz --method pdhg \
```

Comment:
#78 just refactored the instructions to use $HOME everywhere instead of ~

Reply:
Done.